build.gradle

```diff
@@ -42,87 +48,162 @@ task exportProduct{
 	}
 }
 
-task createRestorePoint {
-	doLast {
-		String rp = createRP(SOR_URL);
-		def rpFile = new File("restorepoint.txt");
-		rpFile.write(rp)
+task createRestorePoint(type: org._10ne.gradle.rest.RestTask) {
+	group 'Pega'
+	description 'Create restore point.'
+
+	def targetURL = findProperty("targetURL")
+	uri = "${targetURL}/prweb/PRRestService/SystemManagement/V2/RestorePoint"
```

Comment:
Yeah, I think you are right - just so I am clear, you mean make that part of the "targetURL". Right?